ursula_cli/shell.py

```diff
@@ -331,7 +331,7 @@ def _run_heat(args, hot):
 """
 
     # write out private key if using one generated by heat
-    if private_key != "":
+    if private_key != "" and private_key is not None:
```

Comment:
Why not just `if not private_key:`

Python treats `""` and `None` as boolean False.